errors.js

```diff
@@ -1,5 +1,11 @@
 exports.handleCustomErrors = (err, req, res, next) => {
-  res.status(400).send({ msg: "Error: 400 bad request" }).next(err);
+  if ((err.code = "22P02")) {
+    res.status(400).send({ msg: "Error: 400 bad request" }).next(err);
+  } else if (err.status) {
```

Comment:
if this is for your custom errors, id also check for err.msg -  alot of errors will have a status key, but only your custom errors will have .msg key